internal/handler/job/status/update_test.go

```diff
@@ -143,6 +144,37 @@ func TestUpdateJobStatusNotAllowed(t *testing.T) {
 	assert.Nil(t, status)
 }
 
+func TestUpdateJob_NotifySubscribers(t *testing.T) {
+	db := newInMemoryDB(t)
+	wf := createDirectWorkflow(t, db)
+	job, err2 := db.CreateJob(context.Background(), &model.Job{
```

Comment:
Hm, why is this `err2`? Couldn't it be `err` as that information is acted upon right after this statement... ?